### DISCUSSION TITLE
手册中入门部分内容的一点小建议

### DISCUSSION THREAD

**Discussion by Arxhexa** [hooke007/mpv_PlayKit#273] [mpv 通用]
个人认为 `必要参数` 中可以考虑强调一下 `target-prim` ，现在广色域的显示器越来越多，这个参数理应根据自己屏幕进行调整。
虽然仓库里 `mpv.conf` 写了广色域应该自己手动指定，但可能是我刚开始接触的时候不太熟悉没注意到，然后就直到最近才意识到这点......

意识到这个事是因为试了下inverse tone mapping，本来就是实验性质没什么文档，翻了几个issue里primaries全用的bt.2020，我照着试了下发现画面明显欠饱和（因为我屏幕是99%p3），primaries改对之后看起来就正常了。

**Comment by hooke007** [comment]
我对色彩管理持开放的态度。
和 --target-prim 相关的一些问题

- 涉及偏向主观喜好的因素，很多人喜欢广色域带来的不自然过饱和。甚至有人给我报告过问题（纠正了色彩显示后他觉得欠饱和，实际是常年看惯了错误的色彩产生的不适应）
- 如果显示器支持SRGB模式那优先使用显示器的此模式；在不更改显示器设置的情况下，还原更准确的色彩应该用 --icc-profile （90%覆盖的p3设备如果也照着 target-prim 做偏差就太大了）

P.S. 当然其它的必要参数现在看也并不完全合理。
